go.mod

```diff
@@ -3,15 +3,14 @@ module github.com/crossplane/provider-gcp
 go 1.13
 
 replace (
-	google.golang.org/api => google.golang.org/api v0.21.0
-	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20200331122359-1ee6d9798940
+	google.golang.org/api => google.golang.org/api v0.35.0
 	google.golang.org/grpc => google.golang.org/grpc v1.28.0
 )
```

Comment:
@lukeweber we are chatting today about removing these constraints which I believe would unblock you here. I think it is fine to drop the no longer supported fields in `GKECluster` because I think they are no-ops when submitted currently.